modules/example/src/smithy4s/example/error/NotFoundError.scala

```diff
@@ -1,18 +1,23 @@
 package smithy4s.example.error
 
-import smithy4s.schema.Schema._
+import smithy4s.Schema
+import smithy4s.Hints
+import smithy4s.schema.Schema.string
+import smithy4s.ShapeId
+import smithy4s.schema.Schema.struct
+import smithy4s.ShapeTag
 
 case class NotFoundError(error: Option[String] = None) extends Throwable {
 }
-object NotFoundError extends smithy4s.ShapeTag.Companion[NotFoundError] {
-  val id: smithy4s.ShapeId = smithy4s.ShapeId("smithy4s.example.error", "NotFoundError")
-
-  val hints : smithy4s.Hints = smithy4s.Hints(
+object NotFoundError extends ShapeTag.Companion[NotFoundError] {
+  val id: ShapeId = ShapeId("smithy4s.example.error", "NotFoundError")
+  
+  val hints : Hints = Hints(
     smithy.api.Error.CLIENT.widen,
     smithy.api.HttpError(404),
   )
-
-  implicit val schema: smithy4s.Schema[NotFoundError] = struct(
+  
```

Comment:
Seems to be extra spaces in a lot of places in the examples

Reply:
resolved